phalcon/Helper/Json.zep

```diff
@@ -0,0 +1,109 @@
+
+/**
+ * This file is part of the Phalcon.
+ *
+ * (c) Phalcon Team <team@phalcon.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Phalcon\Helper;
+
+use InvalidArgumentException;
+
+/**
+ * This class offers a wrapper for JSON methods to serialize and unserialize
+ */
+class Json
+{
+    /**
+     * Decodes a string using `json_decode` and throws an exception if the
+     * JSON data cannot be decoded
```

Comment:
It is default behavior for PHP 7.3:
```php
json_decode($foo, JSON_THROW_ON_ERROR);
```

Reply:
The options defaults to `0` so you have to specify that flag. Also we support (for now) 7.2 and above so for now this will suffice until we deprecate 7.2